python/hsfs/core/feature_view_api.py

```diff
@@ -60,6 +61,16 @@ def post(self, feature_view_obj):
             )
         )
 
+    def update(self, feature_view_obj):
+        headers = {"content-type": "application/json"}
+        return self._client._send_request(
```

Comment:
shouldn't you call the `update_from_response_json` method in order to update the object with the response?

Reply:
We do not need to return anything here actually.